crates/ruma-macros/src/doc/ruma_enum_attributes.md

```diff
@@ -0,0 +1,41 @@
+<!-- Keep this comment so the content is always included as a new paragraph -->
+## Container Attributes
+
+* `#[ruma_enum(rename_all = "rule")]` - Override the string representation of all unit variants
+  by using the given `rule`. By default, the string representation uses the unit variant name,
+  with the same case. This attribute allows to vary the case of the string representation with the
+  following rules. The name of the rule always matches the transformation of the string.
+
+  * `lowercase` - Convert to lowercase. `MyVariant` becomes `myvariant`.
+  * `UPPERCASE` - Convert to uppercase. `MyVariant` becomes `MYVARIANT`.
+  * `camelCase` - Convert the first character to lowercase. `MyVariant` becomes `myVariant`.
+  * `snake_case` - Add a `_` before all uppercase characters except at the start and convert all
+    characters to lowercase. `MyVariant` becomes `my_variant`.
+  * `SCREAMING_SNAKE_CASE` - Add a `_` before all uppercase characters except at the start and
+    convert all characters to uppercase. `MyVariant` becomes `MY_VARIANT`.
```

Comment:
I just checked and we don't even use this (same for `SCREAMING-KEBAB-CASE` which I found most susprising).. Should we maybe remove the ones we don't use?